devito/types/grid.py

```diff
@@ -347,7 +349,7 @@ def __init__(self):
             raise ValueError("SubDomain requires a `name`")
         self._dimensions = None
 
-    def __subdomain_finalize__(self, dimensions, shape, **kwargs):
+    def __subdomain_finalize__(self, dimensions, shape, distributor=None, **kwargs):
```

Comment:
why the optional kwargs None?

Reply:
Yeah, not needed (probably an artifact from when I was playing around with something).